rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -13,6 +13,7 @@
 import time
 import timeit
 
+import configobj
```

Comment:
@viduship could you add this configobj in requirements instead of cephci
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/requirements.txt

Reply:
addressed.